query/src/main/scala/filodb/query/exec/rangefn/RangeFunction.scala

```diff
@@ -67,7 +67,7 @@ trait RangeFunction {
             sampleToEmit: TransientRow,
             queryConfig: QueryConfig): Unit
 }
-
+// scalastyle:off
```

Comment:
What style was violated here? Perhaps we shouldn't use it all?

Reply:
@broneill no more than 11 cases i think, here we have 13

Reply:
@sherali42 instead of turning off the style checker, can we just increase the # of cases allowed?  It's in scalastyle.xml